.github/workflows/check-compact-connect.yml

```
@@ -109,3 +109,31 @@ jobs:
 
       - name: Test backend
         run: "cd backend/compact-connect; bin/run_tests.sh -l all -no"
+
+  # The purchases lambda Python version has to be held back because of its Authorize.net dependency
+  # so we will check it separately
+  TestAndLintPurchases:
+    runs-on: ubuntu-latest
+    steps:
+      # Checks-out the repository under $GITHUB_WORKSPACE
+      - uses: actions/checkout@v2
```

actionlint (1.7.8)

the runner of "actions/checkout@v2" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)